crowdsourcing/serializers/task.py

```diff
@@ -1,17 +1,39 @@
-__author__ = 'elsabakiu, asmita, megha'
+__author__ = 'elsabakiu, asmita, megha,kajal'
 
 from crowdsourcing import models
 from rest_framework import serializers
 
 class TaskSerializer(serializers.ModelSerializer):
-    class Meta:
-        model = models.Task
-        fields = ('module', 'statuses', 'status', 'deleted', 'created_timestamp', 'last_updated')
-        read_only_fields = ('module', 'statuses', 'created_timestamp', 'last_updated')
+  deleted = serializers.BooleanField(read_only=True)
+  status=serializers.IntegerField()
+  created_timestamp = serializers.DateTimeField(read_only=True)
 
+  def create(self, **kwargs):
+    task = models.Task.objects.create(deleted = False,**self.validated_data)
+    worker = validated_data.pop('Worker')
+    for t in task:
+      models.TaskWorker.objects.create(task=t,worker=worker)
```

Comment:
tasks are to be generated when the module is created which makes the CUD obsolete but we will accept this for now